src/pb_utils.h

```diff
@@ -212,23 +212,17 @@ struct ResponseSenderBase {
 struct ResponseSendMessage : ResponseSenderBase {
   bi::managed_external_buffer::handle_t response;
 
-  // GPU Buffers handle
+  // A pointer to GPUBuffersShm object.
```

Comment:
seems like we could keep this the same if the structure was renamed to GPUBuffers - is it still a handle or is that seperate from pointer?

Reply:
By pointer I meant a handle to a GPUBuffersShm. Will update the comment.